File: server/当前版本/server/zlj1/game_server/src/com/imop/lj/gameserver/player/msg/GCPopupPanelEnd.java
package com.imop.lj.gameserver.player.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.GCMessage;
/**
 * 通知客户端登录需要弹出的面板已经都发完了，前台可以开始处理了
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCPopupPanelEnd extends GCMessage{
	

	public GCPopupPanelEnd (){
	}
	

	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.GC_POPUP_PANEL_END;
	}
	
	@Override
	public String getTypeName() {
		return "GC_POPUP_PANEL_END";
	}
}

File: u3u2/Assets/Scripts/app/net/s2c/GCPopupPanelEnd.cs

using System;
namespace app.net
{
/**
 * 通知客户端登录需要弹出的面板已经都发完了，前台可以开始处理了
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCPopupPanelEnd :BaseMessage
{

	public GCPopupPanelEnd ()
	{
	}

	protected override void ReadImpl()
	{


	}
	
	protected override void WriteImpl()
    {
        return;
    }
	
	public override short GetMessageType() 
	{
		return (short)MessageType.GC_POPUP_PANEL_END;
	}
	
	public override string getEventType()
	{
		return PlayerGCHandler.GCPopupPanelEndEvent;
	}
	

}
}